lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -435,10 +390,15 @@ protected function transformReportbackItem($data) {
       'created_at' => $data->created_at,
     ];
 
-    // Get kudos data from comma separated values in data, if any.
-    // @TODO: Implement new method of retrieving Kudos; Kudos::get([$ids])
-    $kudos = $this->getKudos($data->kudos);
-    $output['kudos']['data'] = $kudos ? dosomething_kudos_sort($kudos) : [];
+    try {
+      $kudos = Kudos::get($data->kudos);
```

Comment:
@DFurnes and I have decided to leave as is for now and maybe update down the line if there is a specific need.